keras2onnx/ke2onnx/activation.py

```diff
@@ -4,7 +4,7 @@
 # license information.
 # --------------------------------------------------------------------------
 import keras
-from keras.activations import get as _get_activation
+from keras.activations import get as get_activation
```

Comment:
it's not necessary to change the name after 'as', given this rename method is really a private stuff in this module.

Reply:
done